packages/vscode-extension/src/controls/QuickStart.scss

```diff
@@ -40,8 +40,13 @@
     margin-left: 12px;
 
     h2 {
+      font-family: 'SF Pro Text';
```

Comment:
I don't think this is available on Windows. VSC has its own default settings for font family we should adopt that